stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -269,7 +270,11 @@ internal class PaymentSheetActivity : BasePaymentSheetActivity<PaymentResult>()
 
     private fun setupBuyButton() {
         viewBinding.buyButton.completedAnimation.observe(this) { completedState ->
-            completedState?.paymentIntentResult?.let(::onActionCompleted)
+            completedState?.paymentIntentResult?.let { paymentIntentResult ->
```

Comment:
When else does the completed Animation fire and we wouldn't want to pause a moment?

Reply:
We want to animate while we're waiting for any API call to complete. So that would be confirming a PaymentIntent or creating a PaymentMethod.